ReactWindows/ReactNative/Chakra/Executor/ChakraJavaScriptExecutor.cs

```diff
@@ -296,31 +300,22 @@ private JToken ConvertJson(JavaScriptValue value)
             }
             catch (Exception ex)
             {
-                Debug.WriteLine(
-                    "Error in ChakraExecutor.ConsoleCallback: " + ex.Message);
+                Debug.WriteLine($"Error in ChakraExecutor.ConsoleCallback: {ex.Message}");
             }
 
             return JavaScriptValue.Undefined;
         }
 
         private string Stringify(JavaScriptValue value)
         {
-            switch (value.ValueType)
-            {
-                case JavaScriptValueType.Undefined:
-                case JavaScriptValueType.Null:
-                case JavaScriptValueType.Number:
-                case JavaScriptValueType.String:
-                case JavaScriptValueType.Boolean:
-                case JavaScriptValueType.Object:
-                case JavaScriptValueType.Array:
-                    return ConvertJson(value).ToString(Formatting.None);
-                case JavaScriptValueType.Function:
-                case JavaScriptValueType.Error:
-                    return value.ConvertToString().ToString();
-                default:
-                    throw new NotImplementedException();
-            }
+            JavaScriptValue result;
+            Native.ThrowIfError(Native.JsCallFunction(
+                _stringifyFunction, 
+                new[] { _globalObject, value }, 
+                2, out 
+                result));
+
+            return result.ToString();
```

Comment:
Can you change is back? The stringify function is a JSON.stringify call.  For obvious reasons, this will not work when you call it on a JavaScript Function or Error object, which sometimes people will write to the console.